### ISSUE TITLE
Event site

### ISSUE THREAD

**Issue by martinheidegger** [nodeschool/mumbai#1]
I just recognised that the event site for mumbai only shows the link to gdgmumbai but there is no information on the NodeSchool event (place/time/how to register). I am wondering if there is a better link somewhere?

**Comment by anmolview** [comment]
https://nodejs.org/api/buffer.html
http://nodeschool.io/

hi , you can start with 
npm install -g learnyounode

**Comment by pcbishnoi** [comment]
@martinheidegger you're right. We don't have any info on that front. wondering if you can help me out in setting up nodeschool event. probably a similar link, I can refer to and then can put up more info about event.
Sorry, it's been a long time since you raised this question and I am replying you now :(

**Comment by PatrickHeneise** [comment]
@martinheidegger @anmolview @pcbishnoi I'll be in Mumbai next week and would like to help to bootstrap this event. Are you still around?

**Comment by pcbishnoi** [comment]
Glad to know @PatrickHeneise you will be in Mumbai next week. What's plan for event?

**Comment by PatrickHeneise** [comment]
I contacted meetup organizers and sent out some tweets. I can help to bootstrap NodeSchool as a mentor/co-organizer, give a talk, or we can just meet in a social setting. I don’t really have a plan. Just wanted to let you know that I’m around and would love to help with this. I don’t have contacts in Mumbai, so finding a venue that is available is impossible for me.

**Comment by PatrickHeneise** [comment]
@martinheidegger maybe time to close this? :)
